bash/cfScanner.sh

```diff
@@ -91,11 +91,21 @@ fncSubnetToIP() {
         bytes[2]="$(( k+(iparr[2] & maskarr[2]) ))"
         for l in $(seq 1 $((255-maskarr[3]))); do
           bytes[3]="$(( l+(iparr[3] & maskarr[3]) ))"
-          printf "%d.%d.%d.%d\n" "${bytes[@]}"
+          ip="$(printf "%d.%d.%d.%d" "${bytes[@]}")"
+          IP_LIST+=("$ip")
         done
       done
     done
   done
+
+  # Choose random IP addresses from generated IP list
+  if test -v randomNumber; then
+    IP_LIST=($(shuf -e "${IP_LIST[@]}"))
```

Comment:
Add shuf to the dependencies list in readme

Reply:
Isn't `shuf` part of the `coreutils` package? 🤔 Is this needed?

Reply:
We are not sure about all distributions
You are right it will be find in most of Linux distros but it is good to mention people to know about it and adjust script to alert if it is not exist
To keep the integrity of script and resolve all external dependencies